### PR TITLE
feat(osv): pass GitHub token for DB download 

### PR DESCRIPTION
**Pull request by jporzucek**
## Context
Fixes https://github.com/renovatebot/renovate/discussions/24209 There is also an accompanying PR in https://github.com/renovatebot/osv-offline/pull/569

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — COMMENTED** [review]
needs upstream change first

**Comment by viceice** [comment]
> Hi @viceice, since the upstream has been [merged ](https://github.com/renovatebot/osv-offline/pull/569), can you review that one pls?

checks failed, merged main and make tests successful

**Review by viceice — COMMENTED** [review]
import order is wrong, see comment from CI below.

File: lib/workers/repository/process/vulnerabilities.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ import {
 } from '../../../modules/versioning';
 import { sanitizeMarkdown } from '../../../util/markdown';
 import * as p from '../../../util/promises';
+import { find } from '../../../util/host-rules';
```
Comment (viceice) [inline lib/workers/repository/process/vulnerabilities.ts:20]:
```suggestion
```

File: lib/workers/repository/process/vulnerabilities.ts [hunk 2]
```diff
@@ -17,6 +17,7 @@ import {
 } from '../../../modules/versioning';
 import { sanitizeMarkdown } from '../../../util/markdown';
```
Comment (viceice) [inline lib/workers/repository/process/vulnerabilities.ts:18]:
```suggestion
import { find } from '../../../util/host-rules';
import { sanitizeMarkdown } from '../../../util/markdown';
```

**Review by viceice — COMMENTED** [review]
import order is wrong, see comment from CI below.

**Review by viceice — APPROVED** [review]

**Comment by MPV** [comment]
Did this cause...?
- https://github.com/renovatebot/renovate/discussions/26991

**Pull request merged**